data/0.95.0/schema/ciphertext_election_context.schema.json

```diff
@@ -40,26 +31,34 @@
           "type": "integer"
         },
         "elgamal_public_key": {
-          "$ref": "#/definitions/ElementModP"
+          "title": "Elgamal Public Key",
+          "type": "string"
         },
         "commitment_hash": {
-          "$ref": "#/definitions/ElementModQ"
+          "title": "Commitment Hash",
+          "type": "string"
         },
         "manifest_hash": {
-          "$ref": "#/definitions/ElementModQ"
+          "title": "Manifest Hash",
+          "type": "string"
         },
         "crypto_base_hash": {
-          "$ref": "#/definitions/ElementModQ"
+          "title": "Crypto Base Hash",
+          "type": "string"
         },
         "crypto_extended_base_hash": {
-          "$ref": "#/definitions/ElementModQ"
+          "title": "Crypto Extended Base Hash",
+          "type": "string"
```

Comment:
These should still reference ElementModP and ElementModQ . See the Submitted Ballot format.

Reply:
This file was purely code generated per the instructions in the ticket.  As long as I'm customizing it, I'll put in some descriptions.